Dockerfile.protoc

```diff
@@ -13,9 +13,8 @@ RUN curl --retry 5 -OL https://github.com/protocolbuffers/protobuf/releases/down
     && rm -f $PROTOC_ZIP
 
 # Install Go tools
-RUN go install \
-    github.com/twitchtv/twirp/protoc-gen-twirp@v8.1.0 \
-    google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1 \
-    github.com/magefile/mage@v1.14.0
+RUN go install github.com/twitchtv/twirp/protoc-gen-twirp@v8.1.0
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
+RUN go install github.com/magefile/mage@v1.14.0
```

Comment:
Now we can't install different modules with different versions with the same command:
```
root@c5244beadb12:/go# go install \
    github.com/twitchtv/twirp/protoc-gen-twirp@v8.1.0 \
    google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1 \
    github.com/magefile/mage@v1.14.0
go: google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1: all arguments must refer to packages in the same module at the same version (@v8.1.0)
root@c5244beadb12:/go# go install     github.com/twitchtv/twirp/protoc-gen-twirp@v8.1.0     google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1     github.com/magefile/mage@v1.14.0
go: google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1: all arguments must refer to packages in the same module at the same version (@v8.1.0)
```